shared/modules/transaction.utils.js

```diff
@@ -82,6 +84,32 @@ export function txParamsAreDappSuggested(transaction) {
   );
 }
 
+/**
+ * @param data
```

Comment:
Is it accurate to say that this function takes encoded transaction data and matches it against different token standard ABIs to decode that data properly? Wondering if it'd be good to add documentation for this function for people who aren't familiar. In that vein, should this be renamed? Perhaps something like `getTokenTransactionData`, `getStandardTokenTransactionData`,  `decodeTokenTransactionData`, `decodeStandardTokenTransactionData`, etc.

Reply:
> Is it accurate to say that this function takes encoded transaction data and matches it against different token standard ABIs to decode that data properly?

yes I'll add some documentation. And I think seomthing like `decodeStandardTokenTransactionData` or `parseStandardTokenTransactionData` might be good?

Reply:
Done here: https://github.com/MetaMask/metamask-extension/pull/13535/commits/8c6eda77418a327553bdad6f17c4516df39fdebc